source/modules/task-list.js

```diff
@@ -18,36 +32,53 @@
  * @function
  * @description Closes add-task form
  */
-export function cancelTask() {
+function cancelTask() {
     document.getElementById("task-name").value = "";
     document.getElementById("add-task-form").classList.add("hidden");
+    setSaveFlag(SAVE_ON);
+    setTaskContent(null);
 }
 
 /**
  * @name saveTask
  * @function
  * @description Adds new task to task list
  */
-export function saveTask() {
+function saveTask() {
     let taskNameInput = document.getElementById("task-name");
     let taskList = document.getElementById("task-list");
-    let newTask = createCustomTaskTag(taskNameInput.value);
-    taskList.appendChild(newTask);
+
+    if (SAVE_FLAG === SAVE_ON) { // Save new task
+        if (inputSanitizer(taskNameInput.value)){ // check for input
+            alert("Please enter something!!");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9414198)
```suggestion
            alert('Please enter something!!');
```